activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Replace `db:schema:load_if_ruby` and `db:structure:load_if_sql` with meta task `db:load_schema_or_structure`
```

Comment:
We don't need to mention that `db:schema:load_if_ruby` and `db:structure:load_if_sql` are being removed in the changelog entry, since they weren't part of the public API.

From the user's perspective, this pull request adds a _new_ task for them to use, so I'd write something like this:

> Add `db:load_schema_or_structure` task which loads the database schema from the file format specified by  `config.active_record.schema_format`.